charts/apisix-ingress-controller/README.md

```diff
@@ -185,6 +186,7 @@ The same for container level, you need to set:
 | serviceMonitor.annotations | object | `{}` | @param serviceMonitor.annotations ServiceMonitor annotations |
 | serviceMonitor.labels | object | `{}` | @param serviceMonitor.labels ServiceMonitor extra labels |
 | serviceMonitor.metricRelabelings | object | `{}` | @param serviceMonitor.metricRelabelings MetricRelabelConfigs to apply to samples before ingestion. ref: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#metric_relabel_configs |
+| timezone | string | `""` | timezone is the timezone where apisix uses. For example: "UTC" or "Asia/Shanghai" This value will be set on apisix-ingress-controller container's environment variable TZ. You may need to set the timezone to be consistent with your local time zone, otherwise the apisix-ingress-controller's logs may be used to retrieve event in wrong timezone. |
```

Comment:
```suggestion
| timezone | string | `""` | timezone is the timezone where apisix-ingress-controller uses. For example: "UTC" or "Asia/Shanghai" This value will be set on apisix-ingress-controller container's environment variable TZ. You may need to set the timezone to be consistent with your local time zone, otherwise the apisix-ingress-controller's logs may be used to retrieve event in wrong timezone. |
```